connectors/src/connectors/slack/temporal/workflows.ts

```diff
@@ -228,6 +229,37 @@ export async function syncOneMessageDebounced(
   // call here, which will allow the signal handler to be executed by the nodejs event loop. /!\
 }
 
+export async function memberJoinedChannel(
+  connectorId: string,
+  nangoConnectionId: string,
+  dataSourceConfig: DataSourceConfig,
+  channelId: string,
+  userId: string
+): Promise<void> {
+  const slackAccessToken = await getAccessToken(nangoConnectionId);
+  const myUserId = await whoAmI(slackAccessToken);
+  if (myUserId == userId) {
```

Comment:
Shouldn't this be tested before we start the workflow?

Reply:
Yes you are right, I just wanted to keep the strucutre of only talking to slack in Temporal, but, I will fix that.